ledger/heavy/replica/jetkeeper.go

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package replica
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/utils"
+	"github.com/insolar/insolar/internal/ledger/store"
+)
+
+type JetKeeper interface {
+	Add(insolar.PulseNumber, insolar.JetID) error
+	All(insolar.PulseNumber) ([]insolar.JetID, error)
```

Comment:
This method is not necessary if we update synced jets inside `Add`.

Reply:
done